wallaby.js

```diff
@@ -6,10 +6,10 @@ module.exports = function(_wallaby) {
     files: [
       "src/**/*.ts*",
       "src/**/*.css",
-      "omnisharptest/omnisharpUnitTests/**/*.ts*",
-      "!omnisharptest/omnisharpUnitTests/**/*.test.ts*"
+      "test/omnisharp/omnisharpUnitTests/**/*.ts*",
```

Comment:
After searching for what wallaby.js is, I feel confident we can just delete this file.